### ISSUE TITLE
Variables are not correctly interpolated when loading GQL schema from introspection

### ISSUE THREAD

**Issue by cirelli94** [usebruno/bruno#3680]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

I'm trying to load the GQL schema for the documentation using the `Load from Introspection` button but I got this error:
![image](https://github.com/user-attachments/assets/d5d80888-8ae1-4030-a26d-038e0eecda3b)
I'm using an url with a {{baseUrl}} variable, when I replace the {{baseUrl}} with the read value I don't get the error and it works.

Bruno v1.36.1

### Screenshots/Live demo link

![image](https://github.com/user-attachments/assets/d5d80888-8ae1-4030-a26d-038e0eecda3b)

**Comment by ganesh-bruno** [comment]
Hey @cirelli94,

Thanks for reporting this issue.

I'm able to reproduce it by my end. 
Could you confirm you're using a valid URL and the `{{baseUrl}}` variable is set in the environment with correct value.

**Comment by cirelli94** [comment]
Yes I confirm it, I'm using it querying the api, it's just loading the schema from introspection that doesn't work to me.

**Comment by adamtmorgan** [comment]
+1 on this issue. Made my way here via this bug encounter.

**Comment by newday-webnicer** [comment]
+1, as above

**Comment by maxime-bern** [comment]
+1, same here

**Comment by anusree-bruno** [comment]
The variable interpolation issue occurs specifically when using variables from sources other than Collection Environment Variables (like Global Environment Variables or Collection Variables).

As a workaround for now you can use environment variable at the collection level meanwhile we will get this issue fixed.

**Comment by norbertbakkotp** [comment]
+1

**Comment by norbert-bakk** [comment]
+1, facing the same issue

**Comment by egonzalezgmv** [comment]
+1
